ethsigner/signer/hashicorp/src/test/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommandTest.java

```diff
@@ -57,15 +61,19 @@ private String validCommandLine() {
         + " --signing-key-path="
         + PATH_TO_SIGNING_KEY
         + " --timeout="
-        + FIFTEEN;
+        + FIFTEEN
+        + " --tls-server-trust-store-file="
+        + CLIENT_CERT_PFX
+        + " --tls-server-trust-store-password-file="
+        + CLIENT_CERT_PASSWD;
   }
 
-  private String removeFieldFrom(final String input, final String fieldname) {
-    return input.replaceAll("--" + fieldname + "=.*?(\\s|$)", "");
+  private String removeFieldFrom(final String input, final String fieldName) {
```

Comment:
we should probably remove these two functions, and try and use them from the cmdlinehelpers (even if it means reworking some testSupport-ness.

Reply:
We are now using CmdlineHelpers here. Resolving convo.